.github/workflows/build-container.yml

```diff
@@ -57,8 +57,28 @@ jobs:
             ADDITIONAL_ARGS="-except docker.push,docker.tag"
           fi
 
+          # Handle the base image if provided
+          if [ "z${{ github.event.client_payload.base_image }}" != "z" ]; then
+            # Some label was given via remote call
+            PIPELINE_JAVA_IMAGE="-var base_image=${{ github.event.client_payload.base_image }}"
+          fi
+
+          # Handle the resulting container labels if provided
+          if [ "z${{ github.event.client_payload.labels }}" != "z" ]; then
+            # the labels come in as "label1 label2" but we need to provide them as
+            # '["label1", "label2"]'
+            # So we replace the original values with quoted ones and then wrap in braces
+
+            # Split the passed values into an array
+            array=(${{ github.event.client_payload.labels }})
+            # Print every array entry encased by double quotes
+            tags=$(printf "\"%s\"," "${array[@]}")
```

Comment:
question: Does the trailing comma here cause issues? My understanding is it shouldn't be there (NOT a proper JSON array), but maybe we are lucky and it works for now?

Reply:
Yes, I specifically tested it. It's used in `packer build`, so only need to be a list, so this:
```
packer build -var tags='["label1", "label2",]' ...
```
seems to be a legit run.

Reply:
![image](https://github.com/user-attachments/assets/6f2b2f36-fcf4-4d0d-881c-d3666c7ca9b9)

Reply:
I'm guessing this is due to changes in the HCL2 format. I'm pretty sure the trailing comma would have caused issues in the old JSON format.

Reply:
Yea potentially